src-tauri/src/operations.rs

```diff
@@ -255,6 +265,22 @@ impl ProjectManager for LocalProjectManager {
 
         Ok(Test::new(&test.name, test.kind, &test.content))
     }
+
+    fn save_test(&self, project_name: &str, test_name: &str, new_content: &str) -> io::Result<()> {
+        let project_path = self.project_path(project_name);
+        if !project_path.exists() {
+            return Err(io::Error::new(io::ErrorKind::NotFound, "project not found"));
+        }
+
+        let test_path = get_file_with_basename(&project_path, test_name);
+        match test_path {
+            Some(path) => {
+                fs::write(path, new_content)?;
```

Comment:
should we consider using `create` instead of `write` , so if for any reason the `test_path` doesn't exist we just create it ? 🤔 
probably it doesn't even matter for this week so it should be good as it is